api/sentry/utils.py

```diff
@@ -11,18 +11,35 @@
     "firefox-ios": {
         "title": ":testops-apple: iOS",
         "looker_dashboard_url": (
-            "https://mozilla.cloud.looker.com/dashboards/2381"
+            "https://mozilla.cloud.looker.com/dashboards/"
+            "2667?Sentry+Project+ID=6176941&Created+Month=30+days"
         ),
         "confluence_report_url": (
             "https://mozilla-hub.atlassian.net/wiki/spaces/"
             "MTE/pages/1631911951/iOS+Health+Monitor+Report"
         )
     },
     "fenix": {
-        "title": ":testops-android: Android"
+        "title": ":testops-android: Android",
+        "looker_dashboard_url": (
+            "https://mozilla.cloud.looker.com/dashboards/"
+            "2667?Sentry+Project+ID=6375561&Created+Month=30+days"
+        ),
+        "confluence_report_url": (
+            "https://mozilla-hub.atlassian.net/wiki/spaces/"
+            "MTE/pages/1695154291/Android+Health+Monitor+Report"
+        )
     },
     "fenix-beta": {
-        "title": ":testops-android: Android (Beta)"
+        "title": ":testops-android: Android (Beta)",
+        "looker_dashboard_url": (
+            "https://mozilla.cloud.looker.com/dashboards/"
+            "2667?Sentry+Project+ID=6295551&Created+Month=30+days"
+        ),
+        "confluence_report_url": (
+            "https://mozilla-hub.atlassian.net/wiki/spaces/"
+            "MTE/pages/1695154291/Android+Health+Monitor+Report"
```

Comment:
Since I include the Looks for Android Beta in the Android Health Monitor Report confluence page, I have added the confluence URL here.